core/templates/services/UpgradedServices.ts

```diff
@@ -66,7 +66,11 @@ import { CollectionCreationBackendService } from
 import { ComputeGraphService } from 'services/compute-graph.service';
 import { ConceptCardObjectFactory } from
   'domain/skill/ConceptCardObjectFactory';
+import { ConceptCardBackendApiService } from
+  'domain/skill/concept-card-backend-api.service';
 import { ContextService } from 'services/context.service';
+import { UserExplorationPermissionsService } from
```

Comment:
Add these imports in sorted order please.

Reply:
Done